Gems/ROS2/Code/Source/RobotImporter/Utils/RobotImporterUtils.cpp

```diff
@@ -341,161 +342,224 @@ namespace ROS2::Utils
                     }
                 }
+            }
+        }
 
-                return AZStd::nullopt;
-            };
-        };
-
-        constexpr AZStd::string_view PackageSchemePrefix = "package://";
-        UriPrefix packageUriPrefix;
-        packageUriPrefix.m_schemeResolver = GetReplacementSchemeResolver(PackageSchemePrefix,
-            amentPrefixPaths, urdfAncestorPaths, fileExists);
-
-        constexpr AZStd::string_view ModelSchemePrefix = "model://";
-        UriPrefix modelUriPrefix;
-        modelUriPrefix.m_schemeResolver = GetReplacementSchemeResolver(ModelSchemePrefix,
-            amentPrefixPaths, urdfAncestorPaths, fileExists);
 
-        // For a local file path convert the file URI to a local path
-        UriPrefix fileUriPrefix;
-        fileUriPrefix.m_schemeResolver = [](AZ::IO::PathView uriPath) -> AZStd::optional<AZ::IO::Path>
+        // At this point, the path has no identified URI prefix. If it's an absolute path, try to locate and return it.
+        // Otherwise, return an empty path as an error.
+        if (unresolvedPath.IsAbsolute())
         {
-            constexpr AZStd::string_view FileSchemePrefix = "file://";
-            // Paths that start with 'file:///' are absolute paths, so only 'file://' needs to be stripped
-            bool uriPrefixMatchCaseSensitive = true;
-            if (AZ::StringFunc::StartsWith(uriPath.Native(), FileSchemePrefix, uriPrefixMatchCaseSensitive))
+            if (fileExistsCB(unresolvedPath))
             {
-                AZStd::string_view strippedUriPath = uriPath.Native().substr(FileSchemePrefix.size());
-                return AZ::IO::Path(strippedUriPath);
+                AZ_Trace("ResolveAssetPath", R"(Resolved Absolute Path: "%.*s")" "\n", 
+                    AZ_PATH_ARG(unresolvedPath));
+                return unresolvedPath;
             }
-
-            return AZStd::nullopt;
-        };
-
-        // Step 1: Attempt to resolved URI scheme paths
-        // libsdformat seems to convert package:// references to model:// references
-        // So the model:// URI prefix resolver is run first
-        const auto uriPrefixes = AZStd::to_array<UriPrefix>({
-            AZStd::move(modelUriPrefix),
-            AZStd::move(fileUriPrefix),
-            AZStd::move(packageUriPrefix)});
-        for (const UriPrefix& uriPrefix : uriPrefixes)
-        {
-            if (auto resolvedPath = uriPrefix.m_schemeResolver(unresolvedPath);
-                resolvedPath.has_value())
+            else
             {
-                AZ_Printf("ResolveURDFPath", R"(Resolved Path using URI Prefix "%.*s" -> "%.*s")" "\n", AZ_PATH_ARG(unresolvedPath),
-                    AZ_PATH_ARG(resolvedPath.value()));
-                return resolvedPath.value();
+                AZ_Trace("ResolveAssetPath", R"(Failed to resolve Absolute Path: "%.*s")" "\n", 
+                    AZ_PATH_ARG(unresolvedPath));
+                return "";
```

Comment:
repeat: returning curly braces causes the `AZ::IO::Path` default constructor be used over the `const char*` constructor which is minuscule-ly faster.
```suggestion
                return {};
```